### PR TITLE
Desk top version

### PR DESCRIPTION
**Pull request by mumbereh**
I this branch I have added media query to the make the portfolio responsive

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,189 +1,227 @@
...
+                    don’t hestiate to contact me.
+                </p>
+                <h2 class="about-sub-head">Let's Connect</h2>
+                    <ul id="social-media">
+                        <li><a href="#" class="social"><img src="twitter.svg" alt="twitter"></a></li>
+                        <li><a href="#" class="social"><img src="linkdn.png" alt="linkdn-icon"></a></li>
+                        <li><a href="#" class="social"><img src="m.svg" alt="media-con"></a></li>
+                        <li><a href="#" class="social"><img src="hand.svg"alt="github-con"></a></li>
+                        <li><a href="#" class="social"><img src="git.svg"></a></li>
+                    </ul>
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] Kindly adjust  the socail icons on the desktop view to be aligned with the paragrah and text around it.
- Look at the image comparison bellow to have a very detailed view. This gives a better user experince and will enhance your attention to details in replating designs given 👍 

## Your design:
image

## Figma:
image

File: index.html [hunk 2]
```diff
@@ -1,189 +1,227 @@
...
+                    </p>
+                <p class="desk-praa"> 
+                    A smart assistant to make driving more safe, 
+                    efficient, and fun by unlocking your most expensive computer: your car.
+                </p>
+                <ul class="trainning">
+                    <li class="other-skills">html</li>
+                    <li class="ruby">Ruby on rails</li>
+                    <li class="other-skills">css</li>
+                    <li class="other-skills">javascript</li>
```
Comment (Shedrack-Sunday) [inline index.html:154]: - [ ] The skills listed in each of the cards on the figma design are aligned horizontally and well spaced. You can see how they give a very nice visual view and will  definitely draw the attention of a viewer. E.g a hiring manager.
- Kindly adjust yours in that regard. `flexbox` will make implementing that functionality very seamless. 

## Your work:
image

## Figma deisgn:
image

File: index.html [hunk 3]
```diff
@@ -1,189 +1,227 @@
...
             If you like what you see and have a project you need coded,
-            don’t hestiate to contact me.</p>
-            <h2 class="sub-head">Let's Connect</h2>
-            <div class="social-media">
+            don’t hestiate to contact me.
+        </p>
+        <div class="sub-head">
+            <h2>Let's Connect</h2>
+        </div>
+            <div class="social-media">
```
Comment (Shedrack-Sunday) [inline index.html:37]: - [ ] Firstly, kindly adjust the space between the `header` content text and social icons to be a replica of the design given 👍 
- [ ] Secondly, lets adjust the placemnt of the background image. You can see the botton dots is lacking on your design. These chanages might seem minor to you, but as a developer, you're tasked to replicated the works of poduct designsers, of which they take every detail into consideration. In prepario for clients and your mangers work, kindly align the image text properly 👍 
## Your view:
image

## Figma view:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @mumbereh       👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there's one adjustment you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  
- Nice efforts on the UI designs✔️  
  
# Required Changes  🌵 

- [x] You made some impressive implementations which I must say is a job well done. However, there are a few  I suggest   you make In line with the requirements 


_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @mumbereh , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Great design
✅ ... Clean code

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**